src/app/page.tsx

```diff
@@ -1,116 +1,132 @@
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
-      </div>
-    </main>
-  )
+	return (
+		<main>
+			<div className="h-36 w-full">
+				<div className="h-20 w-full flex items-center justify-evenly">
+					<div></div>
+					<div className="font-extrabold text-3xl">
+						<i>Waves</i>
+					</div>
+					<button className="w-32 h-8 bg-blue-950 text-white">
+						Try for Free
+					</button>
+				</div>
+				<div className="h-16 px-28">
+					<div className="">
+						<span className="px-10">Home</span>
+						<span className="px-10">About</span>
+						<span className="px-10">Contact</span>
+					</div>
+				</div>
+			</div>
+			<div className="h-[calc(100vh-9rem)] overflow-hidden">
+				<div className="h-3/5 w-5/6 py-20 px-28">
+					<h2 className="font-extrabold text-6xl mb-12">
+						Censorship-resistant video streaming platform
+					</h2>
+					<p className="w-4/6">
+						<i>
+							Say goodbye to content restrictions and enjoy uninterrupted
+							streaming. Our platform utilizes advanced encryption and
+							decentralized technology to ensure your privacy and freedom of
+							expression. Discover a wide range of diverse and uncensored
+							content from creators worldwide. Join us today and experience true
+							freedom in streaming.
+						</i>
+					</p>
+				</div>
+				<div className="h-2/5 px-28 bg-[url('/assets/wave.svg')] bg-cover mt-5 w-full">
+					<button className="w-32 h-8 mx-4 bg-blue-950 text-white hover:border-2 hover:border-blue-950 hover:bg-white hover:text-black ">
+						Try
+					</button>
+					<button className="w-32 h-8 mx-4 border-2 border-blue-950 hover:bg-blue-950 hover:text-white">
+						Learn more
+					</button>
+				</div>
+			</div>
+			<div className="flex flex-col items-center w-full mt-20">
+				<div className="w-11/12 md:w-9/12 lg:w-6/12 flex items-center my-4">
+					<div className="w-5/6">
+						<h3 className="text-2xl font-bold mb-4">
+							A virtual stage for everything
+						</h3>
+						<p className="text-sm">
+							Step into our virtual world and experience the thrill of live
+							performances, exhibitions, workshops, and so much more, all from
+							the comfort of your own home. No longer limited by physical
+							boundaries, our virtual stage breaks down barriers and brings
+							people together from every corner of the globe.
+						</p>
+					</div>
+					<img
+						src="/assets/undraw1.svg"
+						alt=""
+						className="w-[200px] h-[200px] m-2"
+					/>
+				</div>
+				<div className="w-11/12 md:w-9/12 lg:w-6/12 flex items-center my-4">
+					<img
+						src="/assets/undraw1.svg"
+						alt=""
+						className="w-[200px] h-[200px] m-2"
+					/>
```

Comment:
I would suggest to use `rem`, it way better then pixels

Reply:
I used pixels here because I want this image to be as fixed as possible in terms of size